node/lib/cmd/open.js

```diff
@@ -65,15 +69,67 @@ exports.configureParser = function (parser) {
     parser.addArgument(["path"], {
         type: "string",
         help: "open all submodules at or in 'path'",
-        nargs: "+",
+        nargs: "*",
```

Comment:
The help text is somewhat confusing.  It's trying to account for the following: imagine that you have submodules a/b and a/c.  You could say git meta open a, and that would open all submodules "in" a (meaning, a/b and a/c).  Or you could say git meta open a/b, which would open the submodule "at" a/b (rather than opening any sub-submodules inside of a/b).  That's why there is a disjunction of two prepositions. 

Anyway, if I don't change this, ```$ bin/git-meta open -c HEAD
usage: git-meta open [-h] [-c C] path [path ...]
git-meta open: error: too few arguments
```
even if I add required:false.